### ISSUE TITLE
Match scikit-learn input structure 

### ISSUE THREAD

**Issue by vmasrani** [UBC-MDS/RegscorePy#9]
You should structure your functions to match the input of scikit-learn (i.e. metric(y_true, y_pred, k). This way they will work with pretrained models, and will be easily dropped into existing code. Same goes for R - design your functions so they are flexible and easily interface with the existing ecosystem.

**Comment by simrnsethi** [comment]
This has been implemented.
